server.js

```
@@ -361,7 +508,8 @@ app.get('/api/lookup/:query', async (req, res) => {
             case 'asn':
                 // Remove 'AS' prefix if present
                 const asnNumber = query.replace(/^(AS|as)/i, '');
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)